experiment_manager.py

```diff
@@ -323,7 +330,11 @@ def wait_result(self, job):
         return result
 
     def one_dimensional_job(self, config: Config):
+        # input verification
+        if not len(config.get_iterables()) == 1:
+            raise ValueError("config must have exactly one iterable Parameter")
         variable_param = config.get_iterables()[0]
+        tic()
         circs = []
```

Comment:
What is `circs`?

Reply:
a list of qiskit.QuantumCircuit objects that each one of them is generated by child method get_circ(config)